### PR TITLE
Added warning

### PR DESCRIPTION
**Pull request by psiinon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ The ZAP full scan action runs the ZAP spider against the specified target (by de
 optional ajax spider scan and then a full active scan before reporting the results. The alerts will be maintained as a 
 GitHub issue in the corresponding repository.
 
+Be aware that this action will perform attacks on the taget website.
```
Comment (kingthorin) [inline README.md:10]:
```suggestion
**WARNING** Be aware that this action will perform attacks on the target website.
```

Comment (kingthorin) [reply README.md:10]: Maybe even:
```markdown
## WARNING
Be aware that this action will perform attacks on the target website.
...
```

File: README.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ The ZAP full scan action runs the ZAP spider against the specified target (by de
...
+You should only scan targets that you have permission to test.
+You should also check with your hosting company and any other services such as CNDs that may be affected before running this action.
```
Comment (kingthorin) [inline README.md:12]:
```suggestion
You should also check with your hosting company and any other services such as CDNs that may be affected before running this action.
```

File: README.md (hunk 1 above)
Comment (psiinon) [reply README.md]: +1

File: README.md (hunk 2 above)
Comment (psiinon) [reply README.md]: Good spot

**Comment by kingthorin** [comment]
Thanks

**Comment by thc202** [comment]
Does it also worth mentioning of possible side effects? (e.g. https://www.zaproxy.org/faq/how-can-i-prevent-zap-from-sending-me-1000s-of-emails-via-a-contact-us-form/ )

**Pull request merged**